meetings/2018-02-22.md

```diff
@@ -0,0 +1,160 @@
+# Node.js Foundation Community Committee Meeting 2018-02-22
+
+## Links
+
+* **Recording**: https://youtu.be/uZKB27_Xu4A
+* **GitHub Issue**: https://github.com/nodejs/community-committee/issues/245
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks/edit?usp=sharing
+
+## Present
+- Ben Michel  @obensource
+- Tierney Cyren  @bnb
+- Olivia  @oe
+- Manil  @chowdhurian
+- Jem Bezooyen  @JemBijoux
+- Joe Sepi  @joesepi
+- Michael Dawson @mhdawson 
+- @ZibbyKeaton (Zibby Keaton - observer)
+- @Bamieh (Ahmad Bamieh - observer)
+- @amiller-gh (Adam Miller - observer)
+- @Maurice-Hayward (Maurice Hayward - observer)
+- @tobyfarley (Toby Farley - observer)
+- @ZibbyKeaton (Zibby Keaton - observer)
+
```

Comment:
@waleedashraf  (Waleed Ashraf - observer)
I added myself in docs https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks
at meeting start time.